modules/installation-azure-config-yaml.adoc

```diff
@@ -43,14 +43,14 @@ controlPlane: <2>
     azure:
       osDisk:
         diskSizeGB: 1024 <5>
-      type: Standard_D8s_v3
+      type: Standard_D8s_v3 <5>
   replicas: 3
 compute: <2>
 - hyperthreading: Enabled <3>
   name: worker
   platform:
     azure:
-      type: Standard_D2s_v3
+      type: Standard_D2s_v3 <5>
```

Comment:
This is a remnant of existing documentation, but it might be helpful to follow the same ordering here for `worker` as what is shown in lines 44-36 for `controlPlane`. So something like this (also with a dash in front of the first `hyperthreading` for consistency in the sample):

```
controlPlane: <2>
- hyperthreading: Enabled <3> <4>
  name: master
platform:
    azure:
      osDisk:
        diskSizeGB: 1024 <5>
      type: Standard_D8s_v3 <5>
  replicas: 3
compute: <2>
- hyperthreading: Enabled <3>
  name: worker
  platform:
    azure:
      osDisk:
        diskSizeGB: 512 <5>
       type: Standard_D2s_v3 <5>
```